src/test/java/org/fundacionjala/coding/testvowelcount/VowelsTest.java

```diff
@@ -0,0 +1,48 @@
+package org.fundacionjala.coding.testvowelcount;
+
+import org.fundacionjala.coding.vowelcount.Vowels;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+
+/**
+ * Test vowels.
+ */
+public class VowelsTest {
+    /**
+     * Test case1.
+     */
+    @Test
+    public void testCase1() {
+        final int exp1 = 5;
+        assertEquals("Nope!", exp1, Vowels.getCount("abracadabra"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Nope!" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)